tools/ld-chroma-decoder/comb.h

```diff
@@ -57,6 +57,8 @@ class Comb
         bool showMap = false;
         Decoder::PixelFormat pixelFormat = Decoder::PixelFormat::RGB48;
         bool outputYCbCr = false;
+        bool outputY4m = false;
+        bool isFirstFieldFirst = true;
```

Comment:
You can avoid replicating isFirstFieldFirst here - it'll be the same as videoParameters.isFirstFieldFirst, which is already in this structure.

Reply:
isFirstFieldFirst is a private member of LdDecodeMetaData.

I tried using getIsFirstFieldFirst() originally, but I only have the LdDecodeMetaData::videoParameters struct available.

Reply:
Per our discussion on IRC, this ends up being constant since any potential field swapping will be done on the input to ld-chroma-decode.